### DISCUSSION TITLE
New patch release?

### DISCUSSION THREAD

**Discussion by alexhancock** [ExpediaGroup/graphql-kotlin#1500] [Q&A] [answered]
I'd like to depend on a package version with this fix in place https://github.com/ExpediaGroup/graphql-kotlin/pull/1489. As of now we're depending on `6.0.0-alpha.6` which doesn't have it.

Is there a process or expected timeline for when a version with that change will be cut?

Thank you!

**Comment by dariuszkuc** [comment, marked as answer]
Just published https://github.com/ExpediaGroup/graphql-kotlin/releases/tag/6.1.0

**Comment by alexhancock** [comment]
@dariuszkuc Thank you
